.github/workflows/tests.yml

```diff
@@ -15,22 +15,36 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-      - name: Set up Go 1.19
+      - name: Set up bootstrap Go 1.20
         uses: actions/setup-go@v5.0.0
         with:
-          go-version: 1.19
+          go-version: '1.20'
+          cache: false
         id: go
 
       - name: Check out code into the Go module directory
         uses: actions/checkout@v3.0.2
 
+      - name: Cache downloaded Go version
+        uses: actions/cache@v4
+        with:
+          path: |
+            ~/sdk
+            ~/go/bin/go*
+            ~/go/pkg/mod
+            ~/.cache/go-build
+            ~/.cache/golangci-lint/
+          key: ${{ runner.os }}-go-sdk-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-sdk
+
       - name: Check that source has been prepared
         run: |
           ./tools/prepare-source.sh
           if [ -n "$(git status --porcelain)" ]; then
             echo
-            echo "Please include these changes in your branch: "
-            git status -vv
+            echo "Please include these changes in your branch (only first 5k lines): "
+            git status -vv | head -n5000
```

Comment:
Log of the https://github.com/osbuild/image-builder/pull/1075 PR was over 50MB in size, the diff is not very useful as github UI will not even allow to display such a huge log.